packages/geometry/src/vector3.ts

```diff
@@ -270,6 +273,31 @@ export function angleTo(a: Vector3, b: Vector3): number {
   return Math.acos(theta);
 }
 
+/**
+ * Returns the Euler angle, in radians with the `xyz` ordering, between two vectors.
+ */
+export function eulerTo(a: Vector3, b: Vector3): Euler.Euler {
+  const normalizedA = normalize(a);
+  const normalizedB = normalize(b);
+
+  const dotDelta = Math.cos(Angle.toRadians(1));
+  const dotAB = dot(normalizedA, normalizedB);
+  const vectorsAreParallel = Math.abs(dotAB) > dotDelta;
+
+  if (vectorsAreParallel) {
+    return dotAB > 1 - 1e-6 ? Euler.create() : Euler.create({ x: Math.PI });
+  }
+
+  const normalizedQ = Quaternion.normalize(
+    Quaternion.create({
+      w: 1 + dotAB,
+      ...cross(normalizedA, normalizedB),
+    })
+  );
```

Comment:
Consider adding this as a comment: https://www.xarg.org/proof/quaternion-from-two-vectors/

Reply:
Good thought, added a comment with https://github.com/Vertexvis/vertex-web-sdk/pull/575/commits/dc032586238ce031398eefe205086b8c353c2f7e